plugins/global-search/index.html

```diff
@@ -1,14 +1,13 @@
-<!DOCTYPE html>
-  <html lang="en">
-    <head>
-      <meta charset="UTF-8" />
-      <link rel="icon" type="image/svg+xml" href="/icon.svg" />
-      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-      <title>Global Search</title>
-    </head>
-    <body class="flex flex-col h-screen">
-      <div id="root" class="contents"></div>
-      <script type="module" src="/src/main.tsx"></script>
-    </body>
-  </html>
-  
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/icon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Global Search</title>
+  </head>
+  <body class="flex flex-col h-screen antialiased font-medium">
```

Comment:
I thought we discussed adding antialiased to the default plugin styles?

Reply:
It's in there, but in this project, I didn't import it – mainly for some default styling also applied to the elements that derived from the designed styles